src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/JsonTest/JsonTest.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Data;
+using System.Data.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests
+{
+    public class JsonTest
+    {
+        private readonly ITestOutputHelper _output;
+
+        public JsonTest(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.IsJsonSupported))]
```

Comment:
@David-Engel @cheenamalhotra 

Need some help on the usage of `ConditionalFact` 
I am trying to set this test to run only if JSON is supported. However, this test also runs even if `IsJsonSupported` is set as `false`

Reply:
@deepaksa1Are you using Console or Visual Studio? `xunit.runner.visualstudio` and `xunit.runner.console` work differently. In Visual Studio, you might not be able to run platform-specific tests or certain xUnit extensions as expected. It's better to test using the Console to see the accurate outcome. (This is by my experience. @David-Engel or @cheenamalhotra may have better inputs to help you on this)

Reply:
JRahnama is correct. I've seen VS run tests even when the ConditionalFact says it shouldn't. Console results should behave correctly.

Reply:
Thanks @JRahnama and @David-Engel for pointing it out. I ran the tests from console and the results were as expected.